qdrant-landing/content/articles/binary-quantization-openai.md

```diff
@@ -40,7 +40,7 @@ You can also try out these techniques as described in [Binary Quantization OpenA
 
 ## New OpenAI embeddings: performance and changes
 
-As the technology of embedding models has advanced, demand has grown. Users are looking more for powerful and efficient text-embedding models. OpenAI's Ada-003 embeddings offer state-of-the-art performance on a wide range of NLP tasks, including those noted in [MTEB](https://huggingface.co/spaces/mteb/leaderboard) and [MIRACL](https://openai.com/blog/new-embedding-models-and-api-updates). 
+As the technology of [embedding models](https://qdrant.tech/articles/fastembed/) has advanced, demand has grown. Users are looking more for powerful and efficient text-embedding models. OpenAI's Ada-003 embeddings offer state-of-the-art performance on a wide range of NLP tasks, including those noted in [MTEB](https://huggingface.co/spaces/mteb/leaderboard) and [MIRACL](https://openai.com/blog/new-embedding-models-and-api-updates). 
```

Comment:
Do we include the domain here on purpose?

Just noticed it while skimming this PR.

Reply:
Fixed!

Reply:
:pray: